devmand/gateway/src/devmand/channels/cli/CliFlavour.h

```diff
@@ -37,30 +37,31 @@ class PromptResolver {
   PromptResolver() = default;
   virtual Future<string> resolvePrompt(
       shared_ptr<SessionAsync> session,
-      const string& newline) = 0;
+      const string& newline,
+      shared_ptr<Timekeeper> timekeeper) = 0;
```

Comment:
Why does the PromptResolver need to have TimeKeeper in the API? Isn't it sufficient that DefaultPromptResolver has it in its API?